tests/app/celery/test_reporting_tasks.py

```diff
@@ -429,13 +429,13 @@ def test_create_nightly_notification_status_for_service_and_day(notify_db_sessio
     process_day = datetime.utcnow().date() - timedelta(days=5)
     with freeze_time(datetime.combine(process_day, time.max)):
         create_notification(template=first_template, status='delivered')
-        create_notification(template=second_template, status='temporary-failure')
+        create_notification(template=second_template, status='failed')
 
         # team API key notifications are included
-        create_notification(template=second_template, status='sending', key_type=KEY_TYPE_TEAM)
+        create_notification(template=second_template, status='pending', key_type=KEY_TYPE_TEAM)
 
         # test notifications are ignored
-        create_notification(template=second_template, status='sending', key_type=KEY_TYPE_TEST)
+        create_notification(template=second_template, status='pending', key_type=KEY_TYPE_TEST)
```

Comment:
I see these are changing from `sending` to `pending` statuses, but further up there's a change to swap `pending` for `sending`.

I just wanted to check that these are all lining up as expected!  Is `pending` still a valid status that we're using and it's now just being used differently?

Reply:
Noting that this was taken care of!